addons/point_of_sale/static/src/app/utils/numbers.js

```diff
@@ -0,0 +1,80 @@
+import { roundPrecision } from "@web/core/utils/numbers";
+import { Base } from "@point_of_sale/app/models/related_models";
+
+export const LT = -1;
+export const EQ = 0;
+export const GT = 1;
+
+export function comp(a, b, { precision = 1, method = "HALF-UP" } = {}) {
+    a = roundPrecision(a, precision, method);
+    b = roundPrecision(b, precision, method);
+    const d = a - b;
+    if (d === 0 || roundPrecision(d, precision, method) === 0) {
+        return EQ;
+    }
+    return d < 0 ? LT : GT;
+}
+
+function invertMethod(method) {
+    return method === "UP"
+        ? "DOWN"
+        : method === "DOWN"
+        ? "UP"
+        : method === "HALF-UP"
+        ? "HALF-DOWN"
+        : method === "HALF-DOWN"
+        ? "HALF-UP"
+        : method;
+}
+
+export class AbstractNumbers extends Base {
```

Comment:
Can you add the default values, so it's clear that they have to be added in the inheriting class? 
```suggestion
export class AbstractNumbers extends Base {
    setup() {
        // default values
        this.method = "UP";
        this.precision = Math.pow(10, -2);
    }
```

Reply:
done. introduced as getters.